libs/blocks/global-footer/global-footer.js

```diff
@@ -69,16 +69,38 @@ class Footer {
 
     // TODO: log to LANA if Footer content could not be found
     if (!this.body) return;
-    // TODO: revisit region picker and social links decoration logic
+
     const regionAnchor = this.body.querySelector('.region-selector a');
-    if (regionAnchor?.href) {
-      regionAnchor.setAttribute('href', `${regionAnchor.getAttribute('href')}#_dnt#_dnb`);
+    const regionNextSibling = regionAnchor?.nextSibling;
```

Comment:
This element seems to be empty all the time, maybe we can remove it if not needed.

Reply:
It was just an extra precaution, but not actually needed. I removed it